components/about/TwitterTestimonials.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div
+    class="mx-auto tweet-widget"
+    :class="getClasses(turquoise)"
+  >
+    <div class="tweet-wrapper">
+      <div class="tweet-carousel">
+        <v-carousel
+          v-model="activeTweetIndex"
+          hide-delimiters
+          :show-arrows="!$vuetify.breakpoint.mobile"
+          light
+        >
+          <v-carousel-item v-for="(tweet, i) in tweets" :key="tweet.id">
+            <AboutTweet :tweet="tweets[i]" :turquoise="turquoise" />
+          </v-carousel-item>
+        </v-carousel>
+        <AboutTweetCarouselDelimiter
+          v-show="$vuetify.breakpoint.mobile"
+          :steps="3"
+          :index="activeTweetIndex"
+        />
+      </div>
+      <AboutSvgQuoteSymbol v-if="!turquoise" :width="200" :height="200" class="quote-symbol" />
+    </div>
+  </div>
+</template>
+<script>
+export default {
+  props: {
+    tweets: {
+      type: Array,
+      required: true
+    },
+    turquoise: {
```

Comment:
same as other turqoise comment.

Reply:
Again, I think leaving this as is is actually more in line with the way Vuetify handles these.